### ISSUE TITLE
With 3.1.2: ActionView::Template::Error

### ISSUE THREAD

**Issue by doits** [slim-template/slim-rails#140]
My App works fine with slim-rails 3.1.1, but with the latest upgrade to 3.1.2 it gives me this error on my CI server:

     ActionView::Template::Error:
       Unknown line indicator
         ckeditor/plugins/wsc/dialogs/tmpFrameset, Line 13, Column 0
           {

On my local dev machine I get:

    ActionView::Template::Error (Unknown line indicator
      ckeditor/plugins/preview/preview, Line 11, Column 0
        }

I use slim from master (currently 82be4b5).

**Comment by multpix** [comment]
I also have this error,
temporarily freeze `gem 'slim-rails', '3.1.1'`

**Comment by doits** [comment]
`ckeditor/plugins/preview/preview` has the following content btw:

```html
<script>

// Prevent from DOM clobbering.
if ( typeof window.opener._cke_htmlToLoad == 'string' ) {
	var doc = document;
	doc.open();
	doc.write( window.opener._cke_htmlToLoad );
	doc.close();

	delete window.opener._cke_htmlToLoad;
}

</script>
```

afaik it is only referenced by an `= javascript_include_tag 'application'`, so not sure what happens here.

**Comment by januszm** [comment]
Could you create a sample app on Github with that ckeditor setup so I can clone it and test ?

**Comment by multpix** [comment]
Hi @januszm 
app with error here:
https://github.com/multpix/ckeditor_and_slim

thx

**Comment by januszm** [comment]
@multpix we released version 3.1.3, please re-test
